swag/client/ui/yfu_view.py

```diff
@@ -0,0 +1,272 @@
+import asyncio
+import disnake
+from swag.blocks.swag_blocks import Transaction
+from swag.id import CagnotteId, UserId
+from swag.yfu import Yfu
+from swag.blocks.yfu_blocks import RenameYfuBlock
+
+from .IHS_toolkit import *
+
+##TODO apparition du bouton "Renommer" et "Activer" dynamique
+class YfuNavigation(disnake.ui.View):
+    def __init__(self, swag_client, user_id, first_yfu_id):
+        super().__init__(timeout=None)
+
+        self.swag_client = swag_client
+        self.user_id = user_id
+        self.yfu_ids = sort_yfus_id(
```

Comment:
`sort_yfu_ids`

Reply:
Resolved in 2393786 ("Rename sort_yfus_id by sort_yfu_ids", 2022-02-21)